notebooks/notebook_template_review.py

```diff
@@ -175,14 +176,18 @@ def parse_dir(directory: str) -> None:
             print("\n##", entry.name, "\n")
             parse_dir(entry.path)
         elif entry.name.endswith('.ipynb'):
-            parse_notebook(entry.path)
+            parse_notebook(entry.path, tag=directory.split('/')[-1], linkback=None)
 
 
-def parse_notebook(path: str) -> None:
+def parse_notebook(path: str,
+                   tag: str,
+                   linkback: str) -> None:
     """
         Review the specified notebook.
```

Comment:
Can you elaborate: what does "Review" do?

Reply:
updated